summerofcode/2023.md

```diff
@@ -26,6 +26,46 @@ If you are a project maintainer and consider mentoring during the GSoC 2023 cycl
 
 ## Ideas
 
+### Armada
+
+#### Add Kubectl Plugin for Armada
+
+- Description: Armada is a multi-cluster batch scheduling platform for running high-throughput jobs.  Armada provides a CLI for interacting with jobs, queues and watching events. A common way that Kubernetes users interact with Kubernetes is via kubectl.  This project is to build a kubectl plugin that allows users to submit jobs to Armada using kubectl.
+- Expected outcomes:
+  - Publish a [Krew Plugin](https://krew.sigs.k8s.io/) for Armada
+  - Users can submit jobs, create queues, watch jobs etc using kubectl.
+- Recommend Skills: Go, Kubernetes, Kubectl
+- Mentor(s): Kevin Hannon, @kannon92
```

Comment:
Can you please add your email here @kannon92 ?

Reply:
Done.